test/utils/JourneyRequestHelper.scala

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package utils
+
+import models.DeclarationType
+import models.DeclarationType.DeclarationType
+import models.requests.JourneyRequest
+import org.scalatest.WordSpec
+import views.declaration.spec.UnitViewSpec
+
+trait JourneyRequestHelper extends WordSpec {
```

Comment:
Is this not what @MaciejRewera was talking about doing?

Reply:
I think we have a similar one for the UnitViewSpec

Reply:
I extracted this from UnitViewSpec, because it's useful for models changes too.